pjlib-util/src/pjlib-util/xml.c

```diff
@@ -295,6 +295,7 @@ PJ_DEF(int) pj_xml_print(const pj_xml_node *node, char *buf, pj_size_t len,
 {
     int prolog_len = 0;
     int printed;
+    int print_node;
```

Comment:
I don't really like this convention of declaring all of the variables at the start of the function, but I guess it matches the rest of this function and PJSIP.

print_node is a pretty mediocre name. This value really means 'the length xml_print_node_printed if positive or zero, or an error code if negative'.